web/src/common/relation-graph/FamilyTree.js

```diff
@@ -255,6 +263,7 @@ FamilyTree.propTypes = {
   focusNodeId: PropTypes.number,
   onSkipLengthWeightChange: PropTypes.func,
   onSaveLengthWeightChange: PropTypes.func,
+  onExistEdit: PropTypes.func,
```

Comment:
is that a typo with `exist`, e.g `exit` instead? other than that, the rest of LGTM.

I noted that, for species that don't have length-weight values set at the root. e.g

![image](https://github.com/aodn/nrmn-application/assets/26201635/0fd10f40-6257-462a-ba59-6d812fa311b4)

to be able to use the Family tree editing modal, I had to click `Save observable item` after putting the values into the field, otherwise, the Family tree editing modal will be empty. But I think this is how the users will be happy with, and the root species also not in the scope of the new button `Superseded(by) species`

the feature works as expected

![image](https://github.com/aodn/nrmn-application/assets/26201635/1bdb91af-17d4-41ae-9ac6-f533994892b1)


if it's a type, please update any occurrence, if not, I'm happy to merge this PR.